packages/odyssey-react-mui/src/PasswordField.tsx

```diff
@@ -119,6 +122,20 @@ const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
     const { t } = useTranslation();
     const [inputType, setInputType] = useState("password");
 
+    useEffect(() => {
+      let timeoutId: NodeJS.Timeout;
+
+      // If current inputType is text (cleartext password),
+      // for security set a 30-second timeout to toggle back to password
+      if (inputType === "text") {
+        timeoutId = setTimeout(() => {
+          setInputType("password");
+        }, PASSWORD_VISIBILITY_TIMEOUT);
+      }
+
+      return () => clearTimeout(timeoutId);
+    }, [inputType]);
+
```

Comment:
Should this reset if you type in the input? If so, we need to add `value` to the second arg of `useEffect`.

Reply:
I verified with Gen 2 of the SIW, and regardless of whether the user types in the field or not, the timer is not reset. So this is working as design (for parity).

Reply:
I would think we would want this reset if the user types in the field. Would be weird to be typing and it swap back to being hidden